tests/test_open.py

```diff
@@ -289,6 +290,58 @@ def test_mosaic_mean(self):
                 )
             )
 
+    def test_union_values(self):
```

Comment:
I'm slightly confused, because I thought we recently modified this. In `main` it is 
```python
    def test_union_values(self):
        filenames = [l8_224077_20200518_B2, l8_224078_20200518_B2]
        with gw.open(
            filenames,
            band_names=['blue'],
            mosaic=True,
            bounds_by='union',
        ) as src:
            values = src.values[
                0,
                src.gw.nrows // 2,
                src.gw.ncols // 2 : src.gw.ncols // 2 + 10,
            ]
            self.assertTrue(
                np.allclose(
                    values,
                    np.array(
                        [
                            7524,
                            7538,
                            7573,
                            7625,
                            7683,
                            7661,
                            7643,
                            7773,
                            7697,
                            7566,
                        ]
                    ),
                )
            )
```

Reply:
Sorry I think I started the branch right before you pushed the changes. Sorry about that.

Reply:
Ah, okay, no worries. I can pull in the updated tests from `main`.